helix-core/src/test/java/org/apache/helix/ThreadLeakageChecker.java

```diff
@@ -189,26 +193,26 @@ public static boolean afterClassCheck(String classname) {
         boolean dumpThread = false;
         if (categoryThreadCnt > limit) {
           checkStatus = false;
-          System.out.println(
+          LOG.info(
               "Failure " + threadCategory.getDescription() + " has " + categoryThreadCnt + " thread");
           dumpThread = true;
         } else if (categoryThreadCnt > warningLimit) {
-          System.out.println(
+          LOG.info(
               "Warning " + threadCategory.getDescription() + " has " + categoryThreadCnt + " thread");
           dumpThread = true;
         } else {
-          System.out.println(threadCategory.getDescription() + " has " + categoryThreadCnt + " thread");
+          LOG.info(threadCategory.getDescription() + " has " + categoryThreadCnt + " thread");
         }
         if (!dumpThread) {
           continue;
         }
         // print first 100 thread names
         int i = 0;
         for (Thread t : threadByCat.get(threadCategory)) {
-          System.out.println(i + " thread:" + t.getName());
+          LOG.debug(i + " thread:" + t.getName());
```

Comment:
Nit: LOG.info may be better. Otherwise, it is hard for people using it to see the leaked thread if they wish as they need to turn on debug. which can result in a lot more other lines.

Reply:
This is the main source of verbose output. Before we enforce it, I think it is not a good idea to output it. Note that for now, EVERY single test class prints these 100 lines.